var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -753,67 +751,6 @@ def setup_build_environment(self, env):
         tmp_path = tempfile.mkdtemp(prefix="spack")
         env.set("TEST_TMPDIR", tmp_path)
 
-        env.set("TF_SYSTEM_LIBS", "com_google_protobuf")
-        if spec.satisfies("@:2.3"):
-            # NOTE: INCLUDEDIR is not just relevant to protobuf
-            # see third_party/systemlibs/jsoncpp.BUILD
-            env.set("INCLUDEDIR", spec["protobuf"].prefix.include)
-
-    def patch(self):
-        filter_file(
-            '"-U_FORTIFY_SOURCE",',
-            '"-U_FORTIFY_SOURCE", "-I%s",' % self.spec["protobuf"].prefix.include,
-            "third_party/gpus/crosstool/BUILD.rocm.tpl",
-        )
-        if self.spec.satisfies("@2.12:"):
-            filter_file(
-                'genproto_deps.append("@com_google_protobuf//:well_known_types_py_pb2_genproto")',
-                "pass",
-                "tensorflow/tsl/platform/default/build_config.bzl",
-                string=True,
-            )
-        if self.spec.satisfies("@2.11:"):
-            filter_file(
-                "deps = protodeps + well_known_proto_libs(),",
-                "deps = protodeps,",
-                "tensorflow/tsl/platform/default/build_config.bzl",
-                string=True,
-            )
-        if self.spec.satisfies("@2.3:2.10"):
-            filter_file(
-                "deps = protodeps + well_known_proto_libs(),",
-                "deps = protodeps,",
-                "tensorflow/core/platform/default/build_config.bzl",
-                string=True,
-            )
-        if self.spec.satisfies("@2.4.0:2.5"):
-            text = """
-def protobuf_deps():
-    pass
-"""
-            with open("third_party/systemlibs/protobuf_deps.bzl", "w") as f:
-                f.write(text)
-
-            if self.spec.satisfies("@2.5.0"):
-                file_to_patch = "tensorflow/workspace2.bzl"
-            else:
-                file_to_patch = "tensorflow/workspace.bzl"
-
-            filter_file(
-                '"//third_party/systemlibs:protobuf.bzl": "protobuf.bzl",',
-                '"//third_party/systemlibs:protobuf.bzl": "protobuf.bzl",\n'
-                '"//third_party/systemlibs:protobuf_deps.bzl": "protobuf_deps.bzl",',  # noqa: E501
-                file_to_patch,
-                string=True,
-            )
-
-        # Set protobuf path
-        filter_file(
-            r"(^build:linux --define=PROTOBUF_INCLUDE_PATH=).*",
-            r"\1{0}".format(self.spec["protobuf"].prefix.include),
-            ".bazelrc",
-        )
```

Comment:
Every single release has seen issues with protobuf. Instead of fighting it, I think we should just let bazel build its own protobuf.